src/test/java/org/mifos/integrationtest/cucumber/GSMAStepdef.java

```diff
@@ -0,0 +1,125 @@
+package org.mifos.integrationtest.cucumber;
+
+import io.cucumber.java.en.*;
+import io.restassured.*;
+import io.restassured.builder.*;
+import io.restassured.specification.*;
+import org.mifos.connector.common.gsma.dto.*;
+import org.mifos.integrationtest.common.*;
+
+
+public class GSMAStepdef extends BaseStepDef{
+
+    GSMATransaction gsmaTransaction;
+
+    @Given("I can create GSMATransactionDTO")
+    public void iCanCreateGSMATransactionDTO() {
+        gsmaTransaction = new GSMATransaction();
```

Comment:
creating a DTO can be moved to a static functions. That static functions can be reused in this step definition.